apps/condo/domains/acquiring/schema/SumPaymentsService.test.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * Generated by `createservice acquiring.SumPaymentsService`
+ */
+
+const Big = require('big.js')
+const { makeClient } = require('@condo/keystone/test.utils')
+
+const { makePayer, createTestPayment } = require('@condo/domains/acquiring/utils/testSchema')
+const { SUM_PAYMENTS_QUERY } = require('@condo/domains/acquiring/gql')
+const { makeClientWithSupportUser, makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
+const { createTestOrganizationEmployeeRole, createTestOrganizationEmployee } = require('@condo/domains/organization/utils/testSchema')
+ 
+describe('SumPaymentsService', () => {
+    describe('logic and correct summing', () => {
+        test('admin: sum one payment', async () => {
+            const { admin, billingReceipts, acquiringContext, organization } = await makePayer()
+            const [payment] = await createTestPayment(admin, organization, billingReceipts[0], acquiringContext)
+            const where = { organization: { id: organization.id } }
+            const {
+                data: {
+                    result: {
+                        sum,
+                    },
+                },
+            } = await admin.query(SUM_PAYMENTS_QUERY, { where: where })
+
+            expect(Big(sum).eq(payment.amount)).toBeTruthy()
+        })
+        test('admin: sum 200 payments', async () => {
+            const { admin, billingReceipts, acquiringContext, organization } = await makePayer(200)
+            let totalSum = Big(0)
+            for (let i = 0; i < 200; i++){
```

Comment:
It's too match %) could you please decrease the number of created payments. It's better to move this logic to helper function.